### DISCUSSION TITLE
Update atari 800 Emulator to lastest Version

### DISCUSSION THREAD

**Discussion by thesnable** [retrofw/retrofw.github.io#53] [Emulators and Ports]
Hi!

Is it possible to update the opk of the atari 800 emulator to the lastest Version? (4.2.0)

2.0.2 linked here is rather old.

(https://github.com/atari800/atari800/blob/master/README.TXT)

Thanks alot!

**Comment by Poligraf** [comment]
The retroarch atari800 core for retrofw will  stay up to date from version 1.9.8 (to be released end of august 2021) and is officially supported by retroarch.

The nightly retrofw cores are available from https://buildbot.libretro.com/nightly/dingux/retrofw-mips32/latest/
